<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ratings</class>
 <widget class="QDialog" name="Ratings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BazzarOne</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/new/prefix1/Background/warning.jpeg);</string>
  </property>
  <widget class="QLabel" name="label_productname">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ebrima</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_details">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_details_price">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>200</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_rating">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>381</y>
     <width>114</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_seller">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic">
   <property name="geometry">
    <rect>
     <x>406</x>
     <y>13</y>
     <width>251</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_proceed">
      <property name="text">
       <string>Proceed</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_skip">
      <property name="text">
       <string>SKIP</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
